### PR TITLE
Add rabbitmq autoscaling

### PR DESCRIPTION
**Pull request by nikochiko**

### CODE REVIEW

**Comment by nikochiko** [comment]
This is in production at the moment.

**Comment by devxpy** [comment]
queue length: 3
cooldownPeriod: 20min

min replica: 2
max replica: 15

File: chart/templates/rabbitmq-autoscaling.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+data:
+  host: "{{ $.Values.commonEnv.CLUSTER_WIDE_BROKER_URL | b64enc }}"
+---
+apiVersion: keda.sh/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: keda-trigger-auth-rabbitmq-conn-new
+  namespace: default
+spec:
+  secretTargetRef:
```
Comment (devxpy) [inline chart/templates/rabbitmq-autoscaling.yaml:16]: Can you use `env` here? The separate `BROKER_URL` and `CLUSTER_WIDE_BROKER_URL` doesn't seem right + updating secrets is always a pain https://keda.sh/docs/2.11/concepts/authentication/#environment-variables

Comment (nikochiko) [reply chart/templates/rabbitmq-autoscaling.yaml]: done!

**Pull request merged**